app/src/main/res/layout/activity_error.xml

```diff
@@ -16,15 +16,15 @@
         android:layout_marginBottom="16dp"
         android:text="TextView"
         android:textColor="#000000"
```

Comment:
the text color should go to colors file in resources. Same for text color for login